lex/lexer.mll

```diff
@@ -253,14 +287,19 @@ and comment = parse
       string lexbuf;
       reset_string_buffer();
       comment lexbuf }
-  | "'"
+  | '{' (['a'-'z' '_'] * as delim) '|'
+    { quoted_string delim lexbuf;
+      comment lexbuf }
+  | '\''
```

Comment:
You change this one from `"'"` to `'\''` but not the one in [action] (line 321). Why?

Reply:
I found it more logical, but apparently `"'"` is more used in this file.